examples/widgets/slider_snap_demo.py

```diff
@@ -0,0 +1,62 @@
+"""
+==========================
+Slider Value Snapping Demo
+==========================
+
+You can snap slider values to a discrete values using either the ``valstep``
+argument.
+
+In this example the Freq slider is constrained to be multiples of pi, and the
+Amp slider uses an array as the ``valstep`` argument to more densely sample
+the first part of its range.
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.widgets import Slider, Button
+
+fig, ax = plt.subplots()
+plt.subplots_adjust(bottom=0.25)
+t = np.arange(0.0, 1.0, 0.001)
+a0 = 5
+f0 = 3
+s = a0 * np.sin(2 * np.pi * f0 * t)
+l, = plt.plot(t, s, lw=2)
+
+axcolor = 'lightgoldenrodyellow'
```

Comment:
```suggestion
widget_color = 'lightgoldenrodyellow'
```
may be a bit more clear. (Yes the original slider demo uses `axcolor` but that does not mean we cannot improve :smile: ).

Reply:
Or even the more explicit `slider_background_color`?

Reply:
Also hmm, looking into this I think the slider docstring has an error. In the note at the end it says 

        Additional kwargs are passed on to ``self.poly`` which is the
        `~matplotlib.patches.Rectangle` that draws the slider knob.  See the
        `.Rectangle` documentation for valid property names (``facecolor``,
        ``edgecolor``, ``alpha``, etc.).

but the poly is actually created using `axhspan` which returns a `Polygon` rather than a rectangle https://matplotlib.org/api/_as_gen/matplotlib.pyplot.axhspan.html#matplotlib.pyplot.axhspan

Should that be addressed here or in a follow up?

Reply:
oh woops. I was a bit confused. The color you highlighted here is passed to plt.axes, but I stand by that docstring being slightly incorrect.